### PR TITLE
Add Recovery UI Text customization to Console

### PR DESCRIPTION
**Pull request by Achintha444**
### Purpose
Add Recovery UI Text customization to Console

### Related Issues
- https://github.com/wso2/product-is/issues/19155

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    return (
+        <div data-componentid={ componentId }>
+            <h2>
+                { i18n(CustomTextPreferenceConstants.TEXT_BUNDLE_KEYS.PASSWORD_RECOVERY.HEADING, "Forgot Password?") }
+            </h2>
+            <div className="segment-form">
+                <form method="post" id="totpForm" className="ui large form otp-form">
+                    <p className="text-center" id="instruction">
+                        { i18n(CustomTextPreferenceConstants.TEXT_BUNDLE_KEYS.PASSWORD_RECOVERY.BODY,
+                            "Don't worry, it happens. We will send you an email to reset your password.") }
```
Comment (pavinduLakshan) [inline apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx:51]: There is an ongoing effort to make these error messages professional. These error messages are gonna be changed under that effort. Better if we can use the new error messages, if they have been finalized already.

cc: @kayathiri4

File: apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    props: PasswordRecoveryFragmentInterface
+): ReactElement => {
+    const { ["data-componentid"]: componentId } = props;
+
+    const { i18n } = useBrandingPreference();
+
+    return (
+        <div data-componentid={ componentId }>
+            <h2>
+                { i18n(CustomTextPreferenceConstants.TEXT_BUNDLE_KEYS.PASSWORD_RECOVERY.HEADING, "Forgot Password?") }
```
[the remark this answers is not in the document]
Comment (Achintha444) [reply apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx:45]: "Forgot Password" is used as a fallback to display if the i18n does not work.

File: apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx (hunk 1 above)
Comment (Achintha444) [reply apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx:51]: This is part of this effort, it was discussed in that issue to bring text customization to recovery UIs. AFAIK the new error messages are not finalized, hence we have to go forward with these values.

File: apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
[the remark this answers is not in the document]
Comment (Achintha444) [reply apps/console/src/features/branding/components/preview/sign-in-box/fragments/password-recovery-fragment.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/5394/commits/5a97ffec326c556cae6ddfda2ad2a42cf69991d1

**Review by pavinduLakshan — APPROVED** [review]

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**